usagov/usagov-2021
# Usagov 1673 fix lazy loaded img

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1673

_Work on this ticket with @liucharlie213._

## Description

- Changed the way we do lazy loading by removing the event listener and using an intersectionObserver
- Removed the lazy load module

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



1. Run bin/init
2. Run bin/drush cim
3. Run docker compose up
4. Go to localhost > inspect > Network > click "img" > reload webpage > scroll down through page slowly and watch images get downloaded as you scroll
5. Also can go localhost > inspect > Lighthouse > run page analyze > double check that "defer offscreen images" does not appear in the problematic audits > check that "defer offscreen images" is included in the "passed audits" drop down
6. Go to localhost/admin/modules and check that the module "Lazy-load" it's not there

### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
7. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
